src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+
+  &__image {
+    width: 160px;
+    padding: 32px 3px 40px;
+    height: 134px;
+  }
+  
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-dark-blue;
+    padding-bottom: 4px;
+  }
+  
+  &__code {
+    font-weight: inherit;
+    font-size: inherit;
+    color: $color-grey;
+    line-height: 14px;
+    padding: 0 68px 16px 0;
+  }
+  
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    &-price {
+      font-size: 12px;
+      font-weight: inherit;
+      line-height: 18px;
+      color: $color-grey;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color-dark-blue;
+    }
```

Comment:
don't do it. make them `&__price-number` and `&__price-price`. and rename the last one